server/controller/passport.js

```diff
@@ -52,3 +53,27 @@ passport.use('signup', new LocalStrategy(((username, password, done) => {
     return done(null, false);
   });
 })));
+passport.use(new GoogleStrategy(
+  {
+    clientID: process.env.GOOGLE_CLIENT_ID,
+    clientSecret: process.env.GOOGLE_CLIENT_SECRET,
+    callbackURL: 'http://localhost:3000/api/auth/google/callback',
+  },
+  ((accessToken, refreshToken, profile, done) => {
+    Auth.findOne({ 'google.id': profile.id })
+      .then((res) => {
+        if (!res) {
+          const auth = new Auth({ 'google.name': profile.displayName, 'google.id': profile.id });
+          auth.save((error) => {
+            if (error) return done(null, false);
+            const profile = new User({ _id: auth._id, name: profile.displayName });
```

Comment:
Change the name of this variable to `user` for example and update the save. Currently it is not saving.